boards/arm/qemu_cortex_m0/nrf_timer_timer.c

```diff
@@ -31,41 +33,131 @@ static uint32_t counter_sub(uint32_t a, uint32_t b)
 
 static void set_comparator(uint32_t cyc)
 {
-	nrf_timer_cc_set(TIMER, 0, cyc & COUNTER_MAX);
+	nrf_timer_cc_set(TIMER, NRF_TIMER_CC_CHANNEL0, cyc & COUNTER_MAX);
+}
+
+static uint32_t get_comparator(void)
```

Comment:
optional: this function, and many others in this file, are only called once and do nothing but call another function.  Consider hoisting the call out and deleting these.

Reply:
I imagine these were placed here for code clarity and with optimization level would just be inlined.